Orange/widgets/data/owpaintdata.py

```diff
@@ -594,6 +596,8 @@ def undo(self):
         self._model.execute(self._undo)
 
     def mergeWith(self, other):
+        # other is another instance of the same class, thus
+        # pylint: disable=protected-access
         if self.id() != other.id():
```

Comment:
@ales-erjavec, isn't `id` always `1`? If I'm not mistaken, this check, the above comment, and method `id` can be removed?

Reply:
The check can be, but the `id` method [should stay.](https://doc.qt.io/qt-5/qundocommand.html#id)